### PR TITLE
Redisstream Source initial support 

### PR DESCRIPTION
**Pull request by aavarghese**
This is an initial support for a "redis-streams" trigger that scales based on the PEL for a specific Consumer Group of a Redis Stream. We have some future work to do for returning ConsumerGroup (currently, [an optional Spec field for RedisStreamSource ](https://github.com/knative-sandbox/eventing-redis/blob/master/source/pkg/apis/sources/v1alpha1/redisstream_types.go#L72) but a required field for Keda's Trigger Specification) info in RedisStreamSource's Status field - when that happens, will do some minor changes to eventing-autoscaler-keda.

### CODE REVIEW

File: pkg/reconciler/source/source.go [hunk 1]
```diff
@@ -115,6 +117,15 @@ func (r *Reconciler) Reconcile(ctx context.Context, key string) error {
 			return err
 		}
 		return r.reconcileAwsSqsSource(ctx, awsSqsSource)
+	case "RedisStreamSource":
+		var redisStreamSource = &redisstreamv1alpha1.RedisStreamSource{}
+		// TODO move scheme register up
```
Comment (zroubalik) [inline pkg/reconciler/source/source.go:122]: what is the reason for this TODO? Can't we do it as part of this PR?

**Review by zroubalik — COMMENTED** [review]
Looking good, just minor questio on the TODO leftover

File: pkg/reconciler/source/source.go (hunk 1 above)
Comment (aavarghese) [reply pkg/reconciler/source/source.go:122]: @zroubalik thank you for reminding. I noticed these Todos for the [other sources too above](https://github.com/knative-sandbox/eventing-autoscaler-keda/pull/40/files/e0db839d390647a2058374d230cdda70cfa4feac#diff-0f1badab04aeb2b9af651f700740b4d9fa70f0b6a58d38c0ec0d7daf90478b1aL102). So wanted to make sure we didn't miss that Todo for Redis, as well. Do you know why that was left in there? I don't think there is anything more to do here for Redis at least. So I can remove it...

Comment (zroubalik) [reply pkg/reconciler/source/source.go:122]: @aavarghese haha, sorry. It must be done by me, but I can't recall why 😄 I think we can remove those TODOs, could you please add that to your PR (all of them). Thanks!

Comment (aavarghese) [reply pkg/reconciler/source/source.go]: No problem. Done. :)

**Pull request merged**